packages/cli/src/databases/entities/User.ts

```diff
@@ -141,4 +141,9 @@ export class User extends WithTimestamps implements IUser {
 			scopeOptions,
 		);
 	}
+
+	toJSON() {
+		const { password, apiKey, mfaSecret, mfaRecoveryCodes, ...rest } = this;
+		return rest;
```

Comment:
Shouldn't the `toJSON` method return a string already?

Reply:
it should, but for some reason when I return `JSON.stringify(rest)`, the result is re-stringified version of the stringified data. I'm not sure what's happening here.

Reply:
Looks like we need to return an object here. 

In node.js
![image](https://github.com/n8n-io/n8n/assets/196144/57898eb7-b81d-4d72-aaab-5b89c3bd25d0)

In Firefox
![image](https://github.com/n8n-io/n8n/assets/196144/691e4457-4d24-47a2-9edf-3c717764e107)